boxoffice/views/admin_report.py

```diff
@@ -49,9 +49,15 @@ def admin_org_report(organization):
     permission='org_admin')
 def tickets_report(item_collection):
     headers, rows = item_collection.fetch_all_details()
+    assignee_url_index = headers.index('assignee_url')
+
     def row_handler(row):
         # localize datetime
         row_list = [v if not isinstance(v, datetime) else format_datetime(localize_timezone(v), format='long', locale=get_locale()) for v in row]
+        # add assignee url
+        access_token = row_list[assignee_url_index]
+        if access_token:
+            row_list[assignee_url_index] = url_for('line_items', access_token=access_token, _external=True)
```

Comment:
At this point one has to stop and ask why the query isn't for SQLAlchemy objects instead of a custom select query. This is incredibly ugly.